logstash-core/src/test/java/org/logstash/ObjectMappersTest.java

```diff
@@ -49,4 +49,9 @@ public void testLog4jOMRegisterRubyBasicObjectSerializersFirst() {
         assertNotNull(found);
         assertTrue("RubyBasicObjectSerializer must be registered before others non-default serializers", found instanceof RubyBasicObjectSerializer);
     }
+
+    @Test
+    public void testStreamReadConstraintsApplication() {
+        ObjectMappers.CONFIGURED_STREAM_READ_CONSTRAINTS.validateIsGlobalDefault();
```

Comment:
Are we just asserting that this is called without error? Not quite following what this test is covering.

Reply:
follow up in https://github.com/elastic/logstash/pull/16884